openvalidators/__init__.py

```diff
@@ -27,5 +27,6 @@
 from . import utils
 from . import weights
 
-__version__ = "1.0.2"
-__spec_version__ = 1003
+__version__ = "1.0.3"
+version_split = __version__.split(".")
+__spec_version__ = (1000 * int(version_split[0])) + (10 * int(version_split[1])) + (1 * int(version_split[2]))
```

Comment:
would it still work if the sub version is >= 10? xd 
eg. 1.0.35

Reply:
No, so v1.35.0 would work, but not 1.0.35

Reply:
fair point, but as the authors and maintainers we should hold ourselves responsible for the conventions we keep.

There should be an agreed upon limit to the sub version fields. If we are going to exceed 10 revisions (0-9) we should use different prefactors such as `10,000, 100, 1`